### PR TITLE
Fix the addressettings parsing issues, add proper object for configuration

### PR DESCRIPTION
**Pull request by mkrutov**
@brusdev , @tlbueno ?

### CODE REVIEW

File: test/misc/addresssettings/addresssettings_default_value_test.go [hunk 1]
```diff
@@ -145,7 +153,8 @@ var _ = ginkgo.Describe("AddressSettingsDefaultValueTest", func() {
 
 		urls, err := brokerDeployer.GetExternalUrls(ExpectedURL, 0)
 		address := urls[0]
-		verifyAddressSettingsBool(address, AddressBit, "defaultLastValueQueue", true, hw)
+		value:= retrieveAddressSettings(address,AddressBit, hw)
+        gomega.Expect(value.DefaultExclusiveQueue).To(gomega.Equal(true))
```
Comment (brusdev) [inline test/misc/addresssettings/addresssettings_default_value_test.go:157]: should `value.DefaultExclusiveQueue` be `value.DefaultExclusiveQueue.DefaultLastValueQueue` ?

Comment (mkrutov) [reply test/misc/addresssettings/addresssettings_default_value_test.go:157]: Oh, you are correct. Only matching in json struct though is LastValueQueue.

**Review by brusdev — APPROVED** [review]

File: pkg/test_helpers/http_request_wrapper.go [hunk 2]
```diff
@@ -51,25 +52,32 @@ func (hw *HttpWrapper) PerformHttpRequest(address string) (string, error) {
...
-	request.SetBasicAuth(hw.Password,hw.User)
+	actualPath, _ := url.QueryUnescape(request.URL.Path)
+	request.URL = &url.URL{
+		Scheme: request.URL.Scheme,
+		Host:   request.URL.Host,
+		Opaque: actualPath,
+	}
 	request.Header = *hw.Header
+	request.SetBasicAuth(hw.Password, hw.User)
+	request.URL.User = url.UserPassword("admin", "admin")
```
Comment (tlbueno) [inline pkg/test_helpers/http_request_wrapper.go:68]: hardcoded user and password. will be better if it is configurable using parameters or config file or at least defined as a constant in the code.

Comment (mkrutov) [reply pkg/test_helpers/http_request_wrapper.go:68]: @tlbueno in this case, password and user is meaningless (ie, broker would accept anything)

File: test/misc/addresssettings/setup.go [hunk 3]
```diff
@@ -47,70 +46,21 @@ var _ = ginkgo.AfterEach(func() {
...
-func verifyAddressSettingsInt(address, AddressBit, itemName string,
-	value int, hw *test_helpers.HttpWrapper) {
+func retrieveAddressSettings(address, AddressBit string, hw *test_helpers.HttpWrapper) test_helpers.Value {
 	domain := strings.Split(address, ".")[0]
 	header := strings.Replace(OriginHeader, "NAME", domain, 1)
 	hw.AddHeader("Origin", header)
-	actualURL := "http://admin:admin@" + address + CallAddress + AddressBit
+	actualURL := "http://" + address + CallAddress + AddressBit
+    log.Logf("url (string): %s", actualURL)
 	hw.WithPassword("admin").WithUser("admin")
```
Comment (tlbueno) [inline test/misc/addresssettings/setup.go:55]: hardcoded user and password. will be better if it is configurable using parameters or config file or at least defined as a constant in the code.

File: test/deployment/resources/setup.go [hunk 4]
```diff
@@ -1,26 +0,0 @@
-package resources
```
Comment (tlbueno) [inline test/deployment/resources/setup.go:1]: why the resource limits testing is getting removed?

File: test/deployment/resources/resources_suite_test.go [hunk 5]
```diff
@@ -1,46 +0,0 @@
-package resources
```
Comment (tlbueno) [inline test/deployment/resources/resources_suite_test.go:1]: why the resource limits testing is getting removed?

File: test/deployment/resources/resource_request_test.go [hunk 6]
```diff
@@ -1,40 +0,0 @@
-package resources
```
Comment (tlbueno) [inline test/deployment/resources/resource_request_test.go:1]: why the resource limits testing is getting removed?

File: test/deployment/resources/resource_limits_test.go [hunk 7]
```diff
@@ -1,40 +0,0 @@
-package resources
```
Comment (tlbueno) [inline test/deployment/resources/resource_limits_test.go:1]: why the resource limits testing is getting removed?

**Pull request merged**